src/components/sequencing/SequenceEditor.svelte

```diff
@@ -203,9 +238,14 @@
         compartmentSeqTooltip.of(sequenceTooltip()),
         EditorView.updateListener.of(debounce(sequenceUpdateListener, 250)),
         EditorView.updateListener.of(selectedCommandUpdateListener),
-        EditorView.updateListener.of(debounce(highlightBlock, 250)),
-        Prec.highest([blockTheme, blockHighlighter]),
-        compartmentSeqAutocomplete.of(indentService.of($sequenceAdaptation.autoIndent())),
+        blockTheme,
+        compartmentSeqHighlighter.of([
+          EditorView.updateListener.of(debounce(seqNHighlightBlock, 250)),
```

Comment:
This could technically also use the new debounced function, but because it's in the `onMount` it's less of a concern. Did you want to change it here too @joswig?

Reply:
Addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/a3fa580ca579ba7730ef57692ba66abb560854b6